js/main.js

```diff
@@ -0,0 +1,17 @@
+$(document).on('ready', function() {
+  console.log('Ready, PLAYER ONE?');
+
+  $(document).on('click', '#submit', function(){
+      // Prevent event submission
+      event.preventDefault();
+      
+      console.log($('#login-form').validate());
+      
+      if ($('form').valid()) {
+        $('form').hide();
+        $('body').append('<h3 class ="center">There are many responsive login pages out there, but this one\'s mine.</h3>');
```

Comment:
This exercise asks you to create an element in memory to add as a success message. A more explicit way of doing this might be:

```javascript
$message = $("<h3 class='center'></h3>");
$message.text("There are many responsive login pages out there, but this one's mine.");
$('body').append($message);
```